jscomp/syntax/tests/printer/comments/expected/expr.res.txt

```diff
@@ -226,10 +226,10 @@ let f = (
   /* c6 */ ~x=?,
 ) => /* c7 */ ()
 
-let multiply = (type /* c-2 */ t /* c-1 */, /* c0 */ m1 /* c1 */, /* c2 */ m2 /* c3 */) => ()
+let multiply = (type /* c-2 */ t /* c-1 */, m1 /* c1 */, /* c2 */ m2 /* c3 */) => ()
```

Comment:
Comment c0 has disappeared

Reply:
Would you open a separate issue about formatting for this example?

Reply:
See #6890